### PR TITLE
feat(Animevietsub): add presence

### PR DESCRIPTION
**Pull request by MinhYz**
## Description 

Adding presence for Animevietsub.cc website

## Acknowledgements
- [ ] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [ ] I linted the code by running `yarn format`
- [ ] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
  

</details>

### CODE REVIEW

File: websites/A/Animevietsub/presence.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
+const presence = new Presence({
+		clientId: "1016991973531451502",
+	}),
+	strings = presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused",
```
Comment (DarkVillager) [inline websites/A/Animevietsub/presence.ts:6]: multilang?

File: websites/A/Animevietsub/presence.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+		presenceData.details = `Đang xem: ${titleArrOne} `;
+		presenceData.state = `Tập: ${
+			document.querySelector<HTMLAnchorElement>(".episode.playing").textContent
+		}`;
+
+		if (video.paused) {
+			delete presenceData.startTimestamp;
+			delete presenceData.endTimestamp;
+		}
+		presence.setActivity(presenceData, true);
```
Comment (DarkVillager) [inline websites/A/Animevietsub/presence.ts:97]:
```suggestion
		presence.setActivity(presenceData);
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/A/Animevietsub/presence.ts (hunk 1 above)
Comment (MinhYz) [reply websites/A/Animevietsub/presence.ts:6]: nah

File: websites/A/Animevietsub/presence.ts [hunk 3]
```diff
@@ -0,0 +1,99 @@
+const presence = new Presence({
+		clientId: "1016991973531451502",
+	}),
+	strings = presence.getStrings({
```
Comment (RisingSunLight42) [inline websites/A/Animevietsub/presence.ts:4]:
```suggestion
	strings = await presence.getStrings({
```

File: websites/A/Animevietsub/presence.ts [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+					: "Không tìm thấy còn cặc - Tập ?"
+			).split(" - "),
+			[startTimestamp, endTimestamp] = presence.getTimestamps(
+				Math.floor(video.currentTime),
+				Math.floor(video.duration)
+			);
+		presenceData.smallImageKey = video.paused ? "pause" : "play";
+		presenceData.smallImageText = video.paused
+			? (await strings).pause
+			: (await strings).play;
```
Comment (RisingSunLight42) [inline websites/A/Animevietsub/presence.ts:84]:
```suggestion
			? strings.pause
			: strings.play;
```

**Review by RisingSunLight42 — CHANGES_REQUESTED** [review]

File: websites/A/Animevietsub/presence.ts (hunk 1 above)
Comment (DarkVillager) [reply websites/A/Animevietsub/presence.ts:6]: > nah

You should add it.

File: websites/A/Animevietsub/presence.ts (hunk 3 above)
Comment (RisingSunLight42) [reply websites/A/Animevietsub/presence.ts:4]: By bad, but you can't use await when it's not in a function. I suggest you to move the func in the UpdateData

**Comment by DarkVillager** [comment]
@MinhYz Don't forget to resolve changes that are oudated/have been resolved.

**Comment by MinhYz** [comment]
> @MinhYz Don't forget to resolve changes that are oudated/have been resolved.

<3

File: websites/A/Animevietsub/presence.ts [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+		else if (curPath.startsWith("/lich-chieu-phim.html"))
+			presenceData.details = "Đang xem lịch chiếu anime";
+		else presenceData.details = "Đang xem trang chủ...";
+		presenceData.startTimestamp = browsingTimestamp;
+		presence.setActivity(presenceData);
+		return;
+	}
+	const [video] = document.querySelectorAll("video");
+
+	if (video && !isNaN(video.duration)) {
```
Comment (EncryptedDev) [inline websites/A/Animevietsub/presence.ts:85]:
```suggestion
	if (!isNaN(video?.duration)) {
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**